integration-test/src/test/java/org/apache/iotdb/subscription/it/local/IoTDBSubscriptionBasicIT.java

```diff
@@ -188,9 +188,11 @@ public void onFailure(final Throwable e) {
     // Check row count
     try {
       // Keep retrying if there are execution failures
-      AWAIT.untilAsserted(() -> Assert.assertEquals(200, rowCount.get()));
-      Assert.assertTrue(commitSuccessCount.get() > lastCommitSuccessCount.get());
-      Assert.assertEquals(0, commitFailureCount.get());
+      AWAIT.untilAsserted(() -> {
+        Assert.assertEquals(200, rowCount.get());
+        Assert.assertTrue(commitSuccessCount.get() > lastCommitSuccessCount.get());
+        Assert.assertEquals(0, commitFailureCount.get());
+      });
```

Comment:
By the way, fix the same issue here https://github.com/apache/iotdb/blob/master/integration-test/src/test/java/org/apache/iotdb/subscription/it/local/IoTDBSubscriptionBasicIT.java#L165...